CHANGELOG.md

```diff
@@ -32,6 +33,8 @@ DOCS
 
 VALIDATOR
 
+- Add tests ([#1975](https://github.com/gfx-rs/naga/pull/1975)) **@JCapucho**
+- Restore error in the case of bad scalar widths. ([#1990](https://github.com/gfx-rs/naga/pull/1990)) **@jimblandy**
```

Comment:
```suggestion
```

mostly an internal change (that adds back the error which was previously part of other releases)